docs/source/VUDRM-token.md

```diff
@@ -39,7 +39,8 @@ The DRM policy is included in the VUDRM token as the encrypted third component.
 | `duration_rental`  | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay rental licenses.                                                               |
 | `duration_lease`   | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay lease licenses.                                                              |
 | `duration_persist` | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay persist licenses.                                                           |
-| `match_content_id` | bool     | `true` or `false`         | Fairplay, Widevine       | Boolean for if the content id in the token should be compared with the content id in the used in the license request; the content id in the license request will be the same as the content id used to generate the keys that encrypted a given piece of content.                                                     
+| `match_content_id` | bool     | `true` or `false`         | Fairplay, Widevine       | Boolean for if the content id in the token should be compared with the content id in the used in the license request; the content id in the license request will be the same as the content id used to generate the keys that encrypted a given piece of content. 
```

Comment:
Repetition of 'in the' ('...with the content id in the used in the licence request')